### ISSUE TITLE
Sanic example background worker

### ISSUE THREAD

**Issue by madisvain** [miguelgrinberg/python-socketio#75]
I'm trying to run the example Sanic app.py that has the background worker that should be emitting events after every 10 seconds.

For me I get no output either from the websockets or shell. How should I go about debugging this issue?

The aiohttp example works as expected with no issues.

**Comment by miguelgrinberg** [comment]
First of all, sanic does not support websocket yet. I actually submitted a pull request that adds websocket, so that'll come soon. For now, you only get long-polling.

I think I found the problem. I ported the aiohttp app without thinking too carefully. I believe the problem is that the app is started on the asyncio default loop, but then when sanic starts it creates its own uvloop based loop and runs there.

I was able to get the bg task to run by moving the `start_background_task` line into the `index()` function, where the default loop has been updated. Give that a try. I will think about a cleaner way to start the background task, in any case.
